Update locale files and views

Nirvanjha2004 opened the pull request:
issue #1021

Nirvanjha2004 committed:
Update locale files and views

Nirvanjha2004 commented:
@marcgc21 @xemyst @sajontahsen @KarinePistili  Can You Check this once. Thanks

sahitya-chandra reviewed (changes requested):
Hey @Nirvanjha2004, can you confirm that you have replaced all the occurrences of `PROFILE` with `profile`?

Nirvanjha2004 commented:
Hi @sahitya-chandra Yes , I have replaced all the occurences of PROFILE with profile and also added the variables where were there in PROFILE to profile.

sahitya-chandra reviewed (approved):
Great work @Nirvanjha2004
